cli/state.rs

```diff
@@ -60,6 +60,7 @@ pub struct ThreadSafeState(Arc<State>);
 
 #[cfg_attr(feature = "cargo-clippy", allow(stutter))]
 pub struct State {
+  pub modules: Arc<Mutex<deno::Modules>>,
```

Comment:
This is kind of gross because ThreadSafeState is already an Arc. But it's not clear how this can be avoided as we need an Arc<Mutex<Modules>> elsewhere.

Reply:
Agreed it's not very nice.